src/js_ast.zig

```diff
@@ -1103,15 +1103,13 @@ pub const Symbol = struct {
 
     remove_overwritten_function_declaration: bool = false,
 
-    /// In debug mode, sometimes its helpful to know what source file
-    /// A symbol came from. This is used for that.
-    ///
-    /// We don't want this in non-debug mode because it increases the size of
-    /// the symbol table.
-    debug_mode_source_index: if (Environment.allow_assert)
-        Index.Int
-    else
-        u0 = 0,
+    /// Used in HMR to decide when live binding code is needed.
+    has_been_assigned_to: bool = false,
+
+    comptime {
+        bun.assert_eql(@sizeOf(Symbol), 88);
```

Comment:
non-blocking: could we make `Symbol` smaller by putting `bool` fields into a packed struct?

Reply:
yes. i bet this struct could be 48 or so without all the bloat in it